RELEASES.md

```diff
@@ -11,12 +11,14 @@
 - Enable expermiental `state-scheme` flag to specify Firewood as a state database.
 - Added prometheus metrics for Firewood if it is enabled and expensive metrics are being used.
 - Disable incompatible APIs for Firewood.
+- Removed deprecated flags `coreth-admin-api-enabled`, `coreth-admin-api-dir`, `tx-regossip-frequency`, `tx-lookup-limit`. Use `admin-api-enabled`, `admin-api-dir`, `regossip-frequency`, `transaction-history` instead.
```

Comment:
`coreth/plugin/evm/config/config.md` should be updated to reflect the removal of these flags. 

Alternatively, I can make these changes in https://github.com/ava-labs/coreth/pull/1095.

Reply:
I think they're already removed from there.